travis/large-requirements.txt

```diff
@@ -11,7 +11,6 @@ onnxmltools==1.4.0;
 onnxruntime==0.3.0;
 mleap==0.16.0
 mxnet==1.5.0
-pandas<=0.23.4
```

Comment:
Without this restriction, the latest version will be installed.

Reply:
https://github.com/mlflow/mlflow/pull/3130/checks?check_run_id=884982725#step:3:486